### PR TITLE
[WIP] Feature/py2neo

### PR DESCRIPTION
**Pull request by mammykins**
Reformat data for import into neo4j
Create node and edge lists
Create cypher script to read in with desired schema
Try to do through Python driver

### CODE REVIEW

**Comment by ff-l** [comment]
Would prefer commits to be more expansive, or squash. not a big deal

File: src/neo4j/gds_style.grass [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    "text-color-internal": "#FFFFFF",
+    "shaft-width": "1px"
+  },
+  "relationship.HAS_SUPPORTING_ORGANISATIONS": {
+    "color": "#2E358B",
+    "border-color": "#2E358B",
+    "text-color-internal": "#FFFFFF",
+    "shaft-width": "1px"
+  }
+}
```
Comment (ff-l) [inline src/neo4j/gds_style.grass]: might as well ass a newline here

File: notebooks/neo4j/query_graph.ipynb [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+   "execution_count": null,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2019-06-03T20:27:00.825604Z",
+     "start_time": "2019-06-03T20:27:00.822412Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "NEO4J_AUTH = os.getenv(\"NEO4J_AUTH\")\n",
```
Comment (ff-l) [inline notebooks/neo4j/query_graph.ipynb]: Did you add this to .envrc?

File: notebooks/neo4j/create_node_edge_lists.ipynb [hunk 3]
```diff
@@ -0,0 +1,2437 @@
...
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "
```
Comment (ff-l) [inline notebooks/neo4j/create_node_edge_lists.ipynb]: no need for the param names + assignment here, but a stylistic thing I guess

Comment (ff-l) [inline notebooks/neo4j/create_node_edge_lists.ipynb]: as said on so, this is a black magic level answer, but alright. For future reference https://stackoverflow.com/a/48532692

File: notebooks/neo4j/query_graph.ipynb (hunk 2 above)
Comment (mammykins) [reply notebooks/neo4j/query_graph.ipynb]: Nope, will do

Comment (mammykins) [reply notebooks/neo4j/query_graph.ipynb]: Added defaults

**Pull request merged**